source/importing-css-assets-and-javascript/index.html.md.erb

```diff
@@ -72,6 +72,23 @@ If you want to override GOV.UK Frontend's styles with your own styles, `@import`
 
 If your project uses GOV.UK Frontend toolkit, GOV.UK Template or GOV.UK Elements, you can [configure GOV.UK Frontend to work with them](/compatibility-mode/).
 
+### Silence deprecation warnings in Dart Sass
+
+If you are using Dart Sass v1.33.0 or greater, you may see deprecation warnings when compiling your Sass:
```

Comment:
Looked up Dart Sass releases, and they omit 'v' or 'version', so I'm suggesting this:

```suggestion
If you're using Dart Sass 1.33.0 or greater, you may see deprecation warnings when compiling your Sass:
```